doc/install/contributing.rst

```diff
@@ -121,6 +121,45 @@ There are many other ways to customize git's behavior; see `configuring git`_
 for more information.
 
 
+GNU Make
+~~~~~~~~
+GNU Make facilitates deploying a package by executing corresponding commands
+from the ``Makefile``. For MNE-Python we have two Makefiles, one in the parent
+directory mainly for testing and one in ``/doc`` for building the documentation.
+
+To check if make is already installed type ::
+
+   $ make
+
+into a terminal and you should see ::
+
+   make: *** No targets specified and no makefile found.  Stop.
+
+If you don't see this or something similar:
+
+.. sidebar::
+   If you get:
+
+   *bash: conda: command not found*
+
+   you need to add
+
+   - ``(Anaconda-Path)``
+   - ``(Anaconda-Path)\Scripts``
+
+   to Windows-PATH.
+
+- For Linux/MacOS, get `GNU Make`_
+- For Windows, you can install make for git BASH (which comes with `git for Windows`_):
+
+   1. Download ``make-(newest version)-without-guile-w32-bin.zip`` from `ezwinports`_
```

Comment:
There is one too many spaces on these lines that make it render as a blockquote:

https://21851-1301584-gh.circle-artifacts.com/0/dev/install/contributing.html#gnu-make

Reply:
Thank you, I am fixing it